diff --git a/build.rs b/build.rs
index b2df05e..522ced8 100644
--- a/build.rs
+++ b/build.rs
@@ -259 +259 @@ mod windows {
-        #[cfg(not(debug_assertions))]
+        #[cfg(not(any(debug_assertions, feature = "runtime_shaders")))]
diff --git a/src/platform/windows/directx_renderer.rs b/src/platform/windows/directx_renderer.rs
index bb82725..1e6f164 100644
--- a/src/platform/windows/directx_renderer.rs
+++ b/src/platform/windows/directx_renderer.rs
@@ -1394 +1394 @@ pub(crate) mod shader_resources {
-    #[cfg(debug_assertions)]
+    #[cfg(any(debug_assertions, feature = "runtime_shaders"))]
@@ -1424 +1424 @@ pub(crate) mod shader_resources {
-        #[cfg(debug_assertions)]
+        #[cfg(any(debug_assertions, feature = "runtime_shaders"))]
@@ -1430 +1430 @@ pub(crate) mod shader_resources {
-            #[cfg(not(debug_assertions))]
+            #[cfg(not(any(debug_assertions, feature = "runtime_shaders")))]
@@ -1434 +1434 @@ pub(crate) mod shader_resources {
-            #[cfg(debug_assertions)]
+            #[cfg(any(debug_assertions, feature = "runtime_shaders"))]
@@ -1451 +1451 @@ pub(crate) mod shader_resources {
-        #[cfg(not(debug_assertions))]
+        #[cfg(not(any(debug_assertions, feature = "runtime_shaders")))]
@@ -1491 +1491 @@ pub(crate) mod shader_resources {
-    #[cfg(debug_assertions)]
+    #[cfg(any(debug_assertions, feature = "runtime_shaders"))]
@@ -1557 +1557 @@ pub(crate) mod shader_resources {
-    #[cfg(not(debug_assertions))]
+    #[cfg(not(any(debug_assertions, feature = "runtime_shaders")))]
@@ -1560 +1560 @@ pub(crate) mod shader_resources {
-    #[cfg(debug_assertions)]
+    #[cfg(any(debug_assertions, feature = "runtime_shaders"))]
