diff --git a/build.rs b/build.rs
index 762e5da..82198e3 100644
--- a/build.rs
+++ b/build.rs
@@ -31,6 +31,7 @@ fn main() {
         // Some definitions are missing from the generated bindings without this
         .clang_macro_fallback()
+        .clang_macro_fallback_build_dir(&out_path)
         // Create callable wrapper functions around SPAs `static inline` functions so they
         // can be called via FFI
         .wrap_static_fns(true)
